README.md

```diff
@@ -158,4 +151,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
[ OPTIONAL] 
- Consider moving these lines of text as they're just a guide for you how to add the license. Now that your license is correct, this text is not really necessary again in your README